src/lib/seam/components/AccessCodeDetails/AccessCodeDetails.tsx

```diff
@@ -244,19 +245,14 @@ const formatDate = (date: string): string =>
 const errorFilter = (
   error: AccessCodeError | DeviceError | ConnectedAccountError
 ): boolean => {
-  if ('is_access_code_error' in error && !error.is_access_code_error)
```

Comment:
@xplato Do you happen to recall why the filter logic looks like it does vs this simplified version I've updated it to here?

I'm sure we had a reason, but maybe it's not relevant now or we made a mistake before?

Reply:
I could be wrong, but I seem to remember that this was implemented around the time that we were reworking some errors. Maybe those error types weren't included by default like they are now? Pretty vague remembrance. Not sure exactly, unfortunately.

Reply:
Pretty sure I was wrong ^

I think it's because the design includes these specific types of errors under access codes, but our API classifies these as device errors. I think this should be re-added (in the default filter), otherwise we will not see those.

Reply:
<img width="1253" alt="Screenshot 2023-10-03 at 4 14 40 PM" src="https://github.com/seamapi/react/assets/87919558/fae029a2-b676-4ba3-a753-0f11d6ab46a2">

^^^ That error will be hidden with the new filter logic.

Reply:
That looks like it may just be a mistake in the fake seed.

Reply:
Whoops, you are right! My bad.

Reply:
Will amend.

Reply:
Introduced here, it seems: https://github.com/seamapi/react/commit/447563bc65d92efbc5d5000056fcbcaf60af891e